test/transactions/multisignature.js

```diff
@@ -94,83 +90,20 @@ describe('multisignature.js', function () {
 	describe('#signTransaction', function () {
```

Comment:
I think the description of `signTransaction` should be like so. In order to account for object returned with two properties: `transaction` and `signature`.

```js
describe('#signTransaction', function () {

	var secret = '123';
	var transaction = lisk.transaction.createTransaction('58191285901858109L', 1000, 'secret');
	var signTransaction = multisignature.signTransaction(transaction, secret);

	it('should return an object', function () {
		(signTransaction).should.be.type('object');
	});

	it('should have a transaction property', function () {
		(signTransaction.transaction).should.be.type('string');
		(signTransaction.transaction).should.be.equal(transaction.id);
	});

	it('should have a signature property', function () {
		var length = 128; // crypto_sign_BYTES length

		(signTransaction.signature).should.be.type('string');
		(signTransaction.signature).should.have.lengthOf(length);
	});
});
```